ow_lander/scripts/path_planning_commander.py

```diff
@@ -30,166 +29,129 @@ class PathPlanningCommander(object):
   def __init__(self):
     super(PathPlanningCommander, self).__init__()
     moveit_commander.roscpp_initialize(sys.argv)
-    robot = moveit_commander.RobotCommander()
-    scene = moveit_commander.PlanningSceneInterface()
-
-    move_arm = moveit_commander.MoveGroupCommander("arm")
-    # move_limbs = moveit_commander.MoveGroupCommander("limbs")
-    # move_grinder = moveit_commander.MoveGroupCommander("grinder")
-    display_trajectory_publisher = rospy.Publisher('/move_group/display_planned_path',
-                                                   moveit_msgs.msg.DisplayTrajectory,
-                                                   queue_size=20)
-    self.arm_move_group = move_arm
-    # self.move_limbs = move_limbs
-    # self.move_grinder = move_grinder
+
+    self.arm_move_group = moveit_commander.MoveGroupCommander("arm")
+    self.limbs_move_group = moveit_commander.MoveGroupCommander("limbs")
+    self.grinder_move_group = moveit_commander.MoveGroupCommander("grinder")
+    self.display_trajectory = rospy.Publisher('/move_group/display_planned_path',
+                                              moveit_msgs.msg.DisplayTrajectory,
+                                              queue_size=20)
 
   # === SERVICE ACTIVITIES - Stow =============================
-  def handle_stow(self, req): 
+  def handle_stow(self, req):
     """
     :type req: class 'ow_lander.srv._Stow.StowRequest'
     """
+    print("Unstow arm activity started")
```

Comment:
Aren't "stow" and "unstow" swapped in these 4 print commands?

Reply:
Yes, you are right. I guess this is what reviews are for :sweat_smile:
I will address it

Reply:
addressed